packages/nitro-protocol/contracts/ETHAssetHolder.sol

```diff
@@ -58,8 +58,8 @@ contract ETHAssetHolder is AssetHolder {
 
         // refund whatever wasn't deposited.
         uint256 refund = amount.sub(amountDeposited);
-        (bool success, ) = msg.sender.call{value: refund}('');
-        require(success, 'Could not refund excess funds');
+        msg.sender.send(refund); // forwards a stipend of only 2300 gas, will not revert on failure
```

Comment:
I am far from expert on this matter, but I wanted to highlight [this](https://consensys.net/diligence/blog/2019/09/stop-using-soliditys-transfer-now/)

Reply:
👍 This is a marginal call that we have discussed with the auditors. On balance the current approach of forwarding all gas was deemed a griefing vector that was worth blocking, despite the downside of "having a hard dependency on gas costs".

Reply:
For reference:

[ _5.3 Gas siphoning attack possible in “transfer all” and “claim all”-related methods (Medium)_](https://consensys.net/diligence/audits/private/ljryv8j1b18epn/)

Note that there is no analogous solution offered in this PR for the `ERC20AssetHolder`, since it is difficult to know how much gas to forward (seeing as it depends on the token contract implementation).